### ISSUE TITLE
Update Size scale Error

### ISSUE THREAD

**Issue by snovos** [VadimDez/ng2-pdf-viewer#226]
On page render getting error **PDFViewer._setScale: '0' is an unknown zoom value**, caused by method **getScale**, since _offsetWidth_ returns 0 even though viewport is visible.

``` <div *ngIf="pdfDoc" class="pdf-viewport-container" [ngClass]="{'hidden': isHidden}">
  <button (click)="onPreviewHide()" class="preview-close-btn" mat-mini-fab>
    <mat-icon>close</mat-icon>
  </button>
  <pdf-viewer [src]="pdfDoc" [original-size]="false" (after-load-complete)="onPdfLoadComplete($event)"></pdf-viewer>
</div>
```
Any thoughts?

**Comment by VadimDez** [comment]
Can you provide any steps to reproduce this error?

**Comment by snovos** [comment]
@VadimDez Just pass invalid url to component [src] property and you will get console error. Make sure zone.js is imported to main module.

P.S - There is pending PR in angular/zone repo, because I dont think there is any workaround on your side. You can close issue I think.

**Comment by VadimDez** [comment]
I did some fix for this in 3.0.4, get that version
you can try to add this to your CSS as well:
```
pdf-viewer {
    display: block;
}
```
